static/prism.less

```diff
@@ -0,0 +1,41 @@
+div.code-toolbar {
+	line-height: 0;
+	> .toolbar {
+		line-height: 0;
+		top: 0 !important;
+		right: 0 !important;
```

Comment:
Are the `!important` necessary? If yes right now, can we use a more specific selector to override the other styles, or are there other styles that should be removed?

Reply:
Fixed, thanks